app/src/main/java/io/multy/ui/fragments/dialogs/DonateDialog.java

```diff
@@ -56,7 +61,11 @@ public void onStart() {
     @OnClick(R.id.button_positive)
     void onPositiveClick(View v) {
         v.setEnabled(false);
-        DonationActivity.showDonation(getContext());
+        int donationCode = 0;
```

Comment:
4 lines can be replaced with
int donactionCode = getArguments() == null ? 0 : getArguments().getInt(Constants.EXTRA_DONATION_CODE, 0);